### DISCUSSION TITLE
Welcome to Rio discussions!

### DISCUSSION THREAD

**Discussion by raphamorim** [raphamorim/rio#3] [Announcements]
<p align="center"><img src="https://raphamorim.io/rio/assets/logo.png" width="100"/></p>

## 👋 Welcome to Rio discussion!

We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by patrickinminneapolis** [comment]
How is it using WebGPU... it must run inside a browser right?  Like an electron app?
